test/views/sections/items/ItemPackagingSealChoiceViewSpec.scala

```diff
@@ -36,23 +36,45 @@ class ItemPackagingSealChoiceViewSpec extends SpecBase with ViewBehaviours {
 
     Seq(ItemPackagingSealChoiceMessages.English).foreach { messagesForLanguage =>
 
-      s"when being rendered in lang code of '${messagesForLanguage.lang.code}'" - {
+      s"when the item packaging is not bulk and being rendered in '${messagesForLanguage.lang.code}'" - {
 
         implicit val msgs: Messages = messages(Seq(messagesForLanguage.lang))
         implicit val request: DataRequest[AnyContentAsEmpty.type] = dataRequest(FakeRequest())
 
         val view = app.injector.instanceOf[ItemPackagingSealChoiceView]
         val form = app.injector.instanceOf[ItemPackagingSealChoiceFormProvider].apply()
```

Comment:
These 3 are duplicated, resulting in multiple view / form instances being created, could you move these into the class scope and remove any duplicates?